src/main/java/org/spideruci/tacoco/reporting/ExecutionDataParser.java

```diff
@@ -102,7 +104,11 @@ private IBundleCoverage analyzeStructure(final ExecutionDataStore data)
 			throws IOException {
 		final CoverageBuilder coverageBuilder = new CoverageBuilder();
 		final Analyzer analyzer = new Analyzer(data, coverageBuilder);
-		analyzer.analyzeAll(classesDirectory);
+
+		for (File classDirectory : this.classesDirectories) {
+			analyzer.analyzeAll(classDirectory);
```

Comment:
remind me, what did this method do again? specifically, how did it impact the exec file?

Reply:
oh nvm ... it actually impacts the json output.